src/App.vue

```diff
@@ -151,7 +151,8 @@ import * as Sentry from '@sentry/browser'
 import { getFeatureFlag, updateLdUser } from '@/utils'
 
 // Components
-import { PaySystemAlert, SbcFeeSummary, SbcFooter, SbcHeader } from '@/components'
+import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
+import { PaySystemAlert, SbcFeeSummary, SbcFooter } from '@/components'
```

Comment:
Isn't this equivalent to the previous code?

Reply:
NVM - I see what you've done.

Why did you do this?

Reply:
Something has changed on the components side, that impacted how we import/exported out of our common index. 
I imagine it's more to do with how we create a reference for it locally, ie `export ( default as ... )`
I don't know how/why at this time. Will look into this some more though in the morning as per the explanation. 
Perhaps there is a better way to do this!

Reply:
Not quite, the import/export reference we created in our index file would no longer work.
I've yet to discover the reason why a direct import resolves but looking closer at this.